charts/oes/values.yaml

```diff
@@ -111,6 +111,14 @@ autopilot:
     tag: v3.3.3-202012161333
     pullPolicy: IfNotPresent
 
+  requests:
+    memory: 2Gi
+    cpu: 1
```

Comment:
Let's put it under resources

e.g.
resources:
  requests:
     memory: 2Gi
     cpu: 1
  limits:
    memory: 2Gi
     cpu: 1
     
 and let's do it the same way for all the services below

Reply:
Changed as per the requirements